client/src/GlobalStyles.js

```diff
@@ -36,7 +36,8 @@ const GlobalStyles = createGlobalStyle`
         
       } 
       font-family:"OxygenRegular";
-      font-size:16px;   
+      font-size:1rem;   
+      color: white;
```

Comment:
would be fun to use variables instead, but you dont have to 

```suggestion
      color: var(--text-primary);
```